build/testing.mk

```diff
@@ -0,0 +1,35 @@
+#
+# Makefile fragment for Testing
+#
+
+
+
+lint: deps
+	@echo "=== $(PROJECT_NAME) === [ lint             ]: Validating source code running $(GOLINTER)..."
+	@$(GOLINTER) run ./...
+
+test: test-only
+test-only: test-unit test-integration
```

Comment:
Can we add a line in the development section of the readme that show how to tag tests as unit or integration?